docs/integrations/_category_.json

```diff
@@ -0,0 +1,10 @@
+{
+  "label": "Integrations",
+  "collapsed": true,
+  "position": 3,
+  "link": {
+    "type": "generated-index",
+    "title": "Integrations",
+    "description": "Integration are a principal part of homarr, it's where you can connect your other apps to interact with them with widgets, Homarr currently integrates with the following types of applications:"
```

Comment:
This is a crazy long sentence. Split it please

Reply:
I don't think I can split it. In the context of where it is I believe it works nicely, since there is not a lot of text in the page